companies/jpmorgan-chase.md

```diff
@@ -0,0 +1,11 @@
+---
+fortune_rank:17
```

Comment:
```suggestion
fortune_rank: 17
```

Reply:
Fixed; I should have noticed this and the other spacing errors.